node/src/p2p.rs

```diff
@@ -86,7 +86,7 @@ const FRAUD_PROOF_HEAD_HEIGHT_THRESHOLD: u64 = 20;
 
 type Result<T, E = P2pError> = std::result::Result<T, E>;
 
-/// Representation of all the errors that can occur when interacting with [`P2p`].
+/// Representation of all the errors that can occur from `P2p` component.
```

Comment:
similar to daser,
```suggestion
/// Representation of all the errors that can occur in `P2p` component.
```
or
```suggestion
/// Representation of all the errors that can be returned from `P2p` component.
```